src/solvers/sat/pbs_dimacs_cnf.cpp

```diff
@@ -219,40 +218,40 @@ propt::resultt pbs_dimacs_cnft::prop_solve()
 
 tvt pbs_dimacs_cnft::l_get(literalt a) const
 {
-  int dimacs_lit=a.dimacs();
+  int dimacs_lit = a.dimacs();
 
   // std::cout << a << " / " << dimacs_lit << "=";
 
-  bool neg=(dimacs_lit < 0);
+  bool neg = (dimacs_lit < 0);
   if(neg)
-    dimacs_lit=-dimacs_lit;
+    dimacs_lit = -dimacs_lit;
 
-  std::set<int>::const_iterator f=assigned.find(dimacs_lit);
+  std::set<int>::const_iterator f = assigned.find(dimacs_lit);
 
   if(!neg)
+  {
+    if(f == assigned.end())
     {
-      if(f==assigned.end())
-        {
-          // std::cout << "FALSE" << "\n";
-          return tvt(false);
-        }
-      else
-        {
-          // std::cout << "TRUE" << "\n";
-          return tvt(true);
-        }
+      // std::cout << "FALSE" << "\n";
+      return tvt(false);
     }
+    else
+    {
+      // std::cout << "TRUE" << "\n";
+      return tvt(true);
+    }
+  }
   else
+  {
+    if(f != assigned.end())
     {
-      if(f!=assigned.end())
-        {
-          // std::cout << "FALSE" << "\n";
-          return tvt(false);
-        }
-      else
-        {
-          // std::cout << "TRUE" << "\n";
-          return tvt(true);
-        }
+      // std::cout << "FALSE" << "\n";
+      return tvt(false);
     }
+    else
+    {
+      // std::cout << "TRUE" << "\n";
+      return tvt(true);
+    }
+  }
```

Comment:
looks like these 25 lines could be replaced by `return tvt(neg == (f == assigned.end()))`